mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -257,15 +259,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	METHOD method_31481 isRemoved ()Z
 	METHOD method_31482 unsetRemoved ()V
 	METHOD method_31483 getFirstPassenger ()Lnet/minecraft/class_1297;
-	METHOD method_32312 getFrozenTicks ()I
```

Comment:
Merge issues? Should these have been removed?

Reply:
huh weird

Reply:
Restored in a6fe16a9c